src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -1763,6 +1710,234 @@ println outFile.text
 
 The complete description of user and global space APIs can be found in the http://doc.activeeon.com/javadoc/latest/org/ow2/proactive/scheduler/task/client/DataSpaceNodeClient.html[DataSpaceNodeClient JavaDoc].
 
+NOTE: The `userspaceapi` and `globalspaceapi` <<_glossary_script_bindings,script bindings>> can also be used in other <<_glossary_additional_task_scripts,scripts>> defined inside a <<_glossary_task,ProActive Task>>.
+To know in which scripts these bindings are available, refer to the <<_variables_quick_reference,variables quick reference>>.
+
+[[_task_synchronization_api]]
+=== Synchronization API
+
+Task synchronization is traditionally handled by the ProActive Scheduler through <<Dependency, Task Dependencies>>, where dependant ProActive Tasks are started only after their parents Tasks completed.
+
+In some cases, *static task dependencies* are not enough to synchronize tasks execution.
+
+This is the case when a task depends on the state of another task executed by a different branch of a workflow, a sibling task or even another job/workflow.
```

Comment:
`by a different branch` --> `in a different branch`?

Reply:
ok